dev/Tiltfile

```diff
@@ -156,8 +156,8 @@ sdf_target = "//bin/sdf:sdf"
 local_resource(
     "sdf",
     labels = ["backend"],
-    cmd = "buck2 build @//mode/release {}".format(sdf_target),
-    serve_cmd = "buck2 run @//mode/release {}".format(sdf_target),
+    cmd = "buck2 build @//mode/debug {}".format(sdf_target),
+    serve_cmd = "buck2 run @//mode/debug {}".format(sdf_target),
```

Comment:
Was this for temporary local dev?

Reply:
I wonder if we can add a `buck2 run dev:up-debug` target in the future...

Reply:
Totally missed that, sorry. But I feel we need more optimizations on rebaser than sdf, so I wonder if we could make sdf compile faster.

It's taking a long time here, everything else compiles quickly

Reply:
Fixed